File: remove_cors.js
const fs = require('fs');
const path = require('path');
const dir = 'src/main/java/com/chips/sales_system/controller';

fs.readdirSync(dir).forEach(file => {
    if (file.endsWith('.java')) {
        const fullPath = path.join(dir, file);
        let content = fs.readFileSync(fullPath, 'utf8');
        content = content.replace(/@CrossOrigin\(origins = "\*", maxAge = 3600\)\r?\n?/g, '');
        fs.writeFileSync(fullPath, content);
    }
});
console.log('Removed @CrossOrigin');

File: remove_cors.py
import os
import glob

controller_dir = r"c:\Users\anand\.gemini\antigravity\scratch\chips_sales_system\backend\src\main\java\com\chips\sales_system\controller"
for filepath in glob.glob(os.path.join(controller_dir, "*.java")):
    with open(filepath, "r", encoding="utf-8") as f:
        content = f.read()
    
    content = content.replace('@CrossOrigin(origins = "*", maxAge = 3600)\n', '')
    content = content.replace('@CrossOrigin(origins = "*", maxAge = 3600)', '')
    
    with open(filepath, "w", encoding="utf-8") as f:
        f.write(content)
print("Removed @CrossOrigin from controllers.")
